### PR TITLE
feat: add nogizaka46 blog

### PR DESCRIPTION
**Pull request by Kasper4649**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/nogizaka46/blog
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

## 说明 / Note

官网不再提供博客 RSS 链接。

### CODE REVIEW

File: lib/v2/nogizaka46/blog.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    ctx.state.data = {
+        allowEmpty: true,
+        title: '乃木坂46 公式ブログ',
+        link: 'https://www.nogizaka46.com/s/n46/diary/MEMBER',
+        item:
+            list &&
+            list.map((item) => ({
+                title: item.title,
+                link: item.link,
+                pubDate: item.date,
```
Comment (TonyRL) [inline lib/v2/nogizaka46/blog.js:20]: Please use parseDate()
docs: https://docs.rsshub.app/joinus/pub-date.html#shi-yong-gong-ju-lei

File: lib/v2/nogizaka46/blog.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        link: 'https://www.nogizaka46.com/s/n46/diary/MEMBER',
+        item:
+            list &&
+            list.map((item) => ({
+                title: item.title,
+                link: item.link,
+                pubDate: parseDate(item.date),
+                author: item.name,
+                description: item.text,
+            })),
```
Comment (TonyRL) [inline lib/v2/nogizaka46/blog.js:24]: Please add trimmed link as guid to prevent https://github.com/DIYgod/RSSHub/issues/9134

Comment (Kasper4649) [reply lib/v2/nogizaka46/blog.js:24]: 2022-02-21 20 39 07@2x

It seems like gid is generated automatically, is it necessary to specify this?

Comment (TonyRL) [reply lib/v2/nogizaka46/blog.js:24]: `guid` will use `link` by default if not specified. 

In https://github.com/DIYgod/RSSHub/issues/9134, it seems the params `ima` or other params from the URL keeps changing and causing duplicated items. As this feed uses a similar API as https://github.com/DIYgod/RSSHub/issues/9134, it is likely that this feed will suffer from the same issue.

**Pull request merged**